dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -331,6 +331,38 @@ public void testHostedEditAndDelete() {
         }
     }
 
+    @Test
+    public void testTableToolAndDagContent() {
+        final ApiClient webClient = getWebClient(USER_2_USERNAME, testingPostgres);
+        WorkflowsApi workflowApi = new WorkflowsApi(webClient);
+
+        Workflow workflow = manualRegisterAndPublish(workflowApi, "DockstoreTestUser2/cwl-gene-prioritization", "", "cwl", SourceControl.GITHUB, "/Dockstore.cwl", true);
+        workflow = workflowApi.refresh((workflow.getId()));
+        WorkflowVersion branchVersion = workflow.getWorkflowVersions().stream().filter(wv -> wv.getName().equals("master")).findFirst().get();
+        WorkflowVersion tagVersion = workflow.getWorkflowVersions().stream().filter(wv -> wv.getName().equals("test")).findFirst().get();
+
+        workflowApi.getTableToolContent(workflow.getId(), branchVersion.getId());
```

Comment:
You should verify that the endpoint is working as well, i.e., just because you have successfully cached the value does not mean the endpoint is returning the correct value (it almost certainly actually is the correct value, but test should validate that).

This comment applies several times in this method.